### PR TITLE
RST 998 run security tests in docker

### PR DESCRIPTION
**Pull request by leoapost**
This PR introduces a platform agnostic test framework based on Make and Docker which makes easy to run tests without any local setups. Tests can run with just few make commands on local systems or on CI systems (like Jenkins).

For now, there's only a setup for running the security tests based on OWASP ZAP. Next steps are to add the unit, feature, performance, accessibility tests.

### CODE REVIEW

File: docker/tests/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      CAPYBARA_SERVER_HOST: security-tests
+    command: ./docker/tests/run_security_tests.sh
+    links:
+      - zap
+    ports:
+      - 3000:3000
+
+  zap:
+    image: owasp/zap2docker-stable
+    command: ./zap.sh -daemon -host 0.0.0.0 -port 8095 -config api.disablekey=true -config 'api.addrs.addr.name=.*' -config 'api.addrs.addr.regex=true'
```
Comment (leoapost) [inline docker/tests/docker-compose.yml]: @AlexaVB I wonder if we need to explicitly include the passive scan rules or zap does that by default?

Comment (AlexaVB) [reply docker/tests/docker-compose.yml]: I'm not totally sure about this. I tried it without and it just hung here:
`security-tests_1  | Waiting for ZAP to start`
I think we should leave it in for now.

Comment (leoapost) [reply docker/tests/docker-compose.yml]: OK, how about we merge it for now and fine tune zap later?

Comment (AlexaVB) [reply docker/tests/docker-compose.yml]: Sounds good to me. Please merge.

**Pull request merged**